runtimes/go/storage/sqldb/db.go

```diff
@@ -326,6 +326,7 @@ func (db *Database) Begin(ctx context.Context) (*Tx, error) {
 // this will be made with backwards compatibility in mind, providing ample notice and
 // time to migrate in an opt-in fashion.
 func Driver[T SupportedDrivers](db *Database) T {
+	db.init()
```

Comment:
was this one meant to be part of this pr? no worries if it should be, just seemed a bit un-related